admin/tests/acceptance/authenticated/users/get-test.js

```diff
@@ -238,11 +240,15 @@ module('Acceptance | authenticated/users/get', function (hooks) {
 
   module('when administrator click on remove authentication method button', function () {
     test('not displays remove link and display unchecked icon', async function (assert) {
-      const expectedAuthenticationMethodCount = 2;
       // given
+      this.intl = this.owner.lookup('service:intl');
+      const expectedAuthenticationMethodCount = 2;
+      const connections = this.intl.t('pages.user-details.navbar.connections');
```

Comment:
Une suggestion pour un nommage qui serait un peu plus indicatif :
```suggestion
      const connectionTabLabel = this.intl.t('pages.user-details.navbar.connections');
```